invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/communities.js

```diff
@@ -0,0 +1,257 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2020-2021 CERN.
+// Copyright (C) 2020-2021 Northwestern University.
+// Copyright (C) 2021 Graz University of Technology.
+// Copyright (C) 2021 New York University.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import {
+  SearchAppFacets,
+  SearchAppResultsPane,
+} from "@js/invenio_search_ui/components";
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import React, { useState } from "react";
+import {
+  Input,
+  Card,
+  Container,
+  Grid,
+  Label,
+  Item,
+  Button,
+  Segment,
+} from "semantic-ui-react";
+import _truncate from "lodash/truncate";
+import {
+  BucketAggregation,
+  SearchBar,
+  Count,
+  Sort,
+  ResultsList,
+  Pagination,
+  ResultsPerPage,
+} from "react-searchkit";
+import {
+  RDMBucketAggregationElement,
+  RDMRecordFacetsValues,
+  SearchHelpLinks,
+} from "../../search/components";
+
+function ResultsGridItemTemplate({ result, index }) {
+  return (
+    <Card fluid key={index} href={`/communities/${result.metadata.id}`}>
```

Comment:
is there any way to avoid hardcoding urls?

Reply:
Hmm at the moment, we don't have a way to inject configuration to the overriden components I think from somewhere central.